src/python/pants/backend/codegen/thrift/apache/python/additional_fields.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+from pants.backend.codegen.thrift.target_types import (
+    ThriftSourcesGeneratorTarget,
+    ThriftSourceTarget,
+)
+from pants.engine.target import StringField
+
+
+# TODO: Maybe merge this with the equivalent field defined in protobuf codegen?
```

Comment:
That's reasonable. It could be defined in `backend/python/target_types.py` in the top section of "common" fields.

Not a big deal either way.

Reply:
Cool, will leave refactor to a separate PR.